src/include/OSL/dual.h

```diff
@@ -71,7 +72,7 @@ class Dual {
     OSL_HOSTDEVICE OIIO_CONSTEXPR14 Dual (const T &x) {
         m_data[0] = x;
         for (int i = 1; i <= PARTIALS; ++i)
-            m_data[i] = T(0.0);
+            m_data[i] = zero();
```

Comment:
why?

Reply:
float3 isn't constructible via `T(0.0)`. At line 1087 is the Cuda specialization that returns `float3{0,0,0}`.

If I remember, tried moving it to `T(0.f)` first, but was still a compile-error.

Reply:
I see, sure.